dashboard/pages/word_cloud_dashboard.py

```diff
@@ -72,9 +94,9 @@ def generate_wordcloud(word_freq: dict, title: str, colormap: str):
     """Generates and returns a word cloud from word frequencies."""
 
     wordcloud = WordCloud(
-        width=800, height=400, background_color="white", colormap=colormap).generate_from_frequencies(word_freq)
+        width=1000, height=500, max_words=100, background_color="white", colormap=colormap).generate_from_frequencies(word_freq)
```

Comment:
increasing the width makes it much easier to read, also good idea to set a max_words